### PR TITLE
zh-cn: lint glossary

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/glossary/browser/index.md [hunk 1]
```diff
@@ -9,7 +11,7 @@ slug: Glossary/Browser
 
 ## 参见
 
-- 维基百科上的[浏览器](https://zh.wikipedia.org/wiki/浏览器)
+- 维基百科上的[浏览器](https://zh.wikipedia.org/wiki/网页浏览器)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/browser/index.md:14]: fix redirect

File: files/zh-cn/glossary/buffer/index.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 title: 缓冲区
 slug: Glossary/Buffer
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-缓冲区是物理内存中的一个存储区域，当数据进行转移时用来临时存放数据。
+缓冲区是物理内存中的一个存储区域，用于在数据进行转移时临时存放数据。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/buffer/index.md:10]: some improvements

File: files/zh-cn/glossary/cache/index.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+title: 缓存
 slug: Glossary/Cache
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-**cache** (web cache 或者 HTTP cache) 是临时存储 HTTP 响应的组件，只要它符合一定条件，就可以用于后续的 HTTP 请求。
+**缓存** (Web cache 或者 HTTP cache) 是临时存储 HTTP 响应的组件。如果满足某些条件，则可以由缓存满足后续请求。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/cache/index.md:10]: 1. give it a name
2. some improments

File: files/zh-cn/glossary/cacheable/index.md [hunk 4]
```diff
@@ -1,17 +1,19 @@
...
 title: 可缓存
 slug: Glossary/Cacheable
+l10n:
+  sourceCommit: 35d631c3a3f4050344ffb8c1bf033137943c10b6
 ---
 
 {{GlossarySidebar}}
 
-**可缓存**的响应是可以缓存的 HTTP 响应，它被存储起来以便后续的检索和使用，省去了对服务器的新的请求。并非所有的 HTTP 响应都可以被缓存，以下是 HTTP 响应被缓存的约束条件：
+**可缓存**的响应是可以缓存的，被存储起来以便后续的检索和使用的 HTTP 响应，省去了对服务器进行新的请求的需要。并非所有的 HTTP 响应都可以被缓存，以下是 HTTP 响应被缓存的约束条件：
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/cacheable/index.md:10]: some improvements

File: files/zh-cn/glossary/cacheable/index.md [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
 
 - 请求中使用的方法本身就是*可缓存的*，即 {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} 方法。如果指示了有效期并且设置了 {{HTTPHeader("Content-Location")}} 标头，{{HTTPMethod("POST")}} 或 {{HTTPMethod("PATCH")}} 请求的响应也可以被缓存，但是这很少被实现。例如，Firefox 就不支持它（[Firefox bug 109553](https://bugzil.la/109553)）。其他方法，如 {{HTTPMethod("PUT")}} 或 {{HTTPMethod("DELETE")}} 是不可缓存的，其结果也不能被缓存。
 - 响应的状态码对应用程序的缓存可知，且被认为是*可缓存的*。以下状态代码是可缓存的：{{HTTPStatus("200")}}、{{HTTPStatus("203")}}、{{HTTPStatus("204")}}、{{HTTPStatus("206")}}、{{HTTPStatus("300")}}、{{HTTPStatus("301")}}、{{HTTPStatus("404")}}、{{HTTPStatus("405")}}、{{HTTPStatus("410")}}、{{HTTPStatus("414")}} 和 {{HTTPStatus("501")}}。
 - 响应中有一些*特定的标头*，如可以防止缓存的 {{HTTPHeader("Cache-Control")}}。
 
-请注意，对特定 URI 的一些不可缓存的请求/响应可能会使同一 URI 上先前缓存的响应失效。例如，对 pageX.html 的 {{HTTPMethod("PUT")}} 将使所有对同一 URI 的 {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} 的缓存请求失效。
+请注意，对特定 URI 的一些不可缓存的请求或响应可能会使同一 URI 上先前缓存的响应失效。例如，对 `/pageX.html` 的 {{HTTPMethod("PUT")}} 将使所有对 `/pageX.html` 的 {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} 的缓存请求失效。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/cacheable/index.md:16]: add missing styles

File: files/zh-cn/glossary/caldav/index.md [hunk 6]
```diff
@@ -1,14 +1,16 @@
...
-CalDAV（{{Glossary("WebDAV")}} 的日历扩展 (Calendaring extensions to WebDAV)）是一种由 {{Glossary("IETF")}} 标准化的{{glossary("protocol", "协议")}}，用于远程访问{{glossary("server", "服务器")}}上的日历数据。
+CalDAV（{{Glossary("WebDAV")}} 的日历扩展，Calendaring extensions to WebDAV）是一种由 {{Glossary("IETF")}} 标准化的{{glossary("protocol", "协议")}}，用于远程访问{{glossary("server", "服务器")}}上的日历数据。
 
 ## 参见
 
 - 维基百科上的 [CalDAV](https://en.wikipedia.org/wiki/CalDAV)
-- [RFC 4791: Calendaring extensions to WebDAV (CalDAV)](https://datatracker.ietf.org/doc/html/rfc4791)
-- [RFC 6638: Scheduling Extensions to CalDAV](https://datatracker.ietf.org/doc/html/rfc6638)
+- [RFC 4791：WebDAV（CalDAV）的日历拓展](https://datatracker.ietf.org/doc/html/rfc4791)
+- [RFC 6638：CalDAV 的安排拓展](https://datatracker.ietf.org/doc/html/rfc6638)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/caldav/index.md:16]: translate name

File: files/zh-cn/glossary/camel_case/index.md [hunk 7]
```diff
@@ -19,6 +19,6 @@ l10n:
 
 ## 参见
 
-- [蛇形命名法](/zh-CN/docs/Glossary/Snake_case)
-- [烤串命名法](/zh-CN/docs/Glossary/Kebab_case)
+- {{Glossary("Snake_case", "蛇形命名法")}}
+- {{Glossary("Kebab_case", "烤串命名法")}}
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/camel_case/index.md:23]: link fixes

File: files/zh-cn/glossary/card_sorting/index.md [hunk 8]
```diff
@@ -1,11 +1,13 @@
...
 title: 卡片分类法
 slug: Glossary/Card_sorting
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-卡片分类法是一种简单的技巧，{{glossary("Information architecture")}} 通常是邀请参与网站开发的设计师（或是开发其他类型产品的人），让他们写下他们认为这个产品应当包含的内容、服务和功能，然后将这些功能分组。一个很好的例子是考虑网站上每个页面应当显示什么样的内容。这个名字源于这个分类是通过把要分类的项目写在卡片上，再通过排列卡片完成的。
+卡片分类法是一种简单的用于{{glossary("Information architecture", "信息架构")}}的技术。在这种技术中，参与网站（或其他类型产品）设计的人被邀请写下他们认为产品应该包含的内容/服务/特性，然后将这些特性组织成类别或分组。例如，这可以用来确定网站的每个页面应该放置什么内容。该名称来源于通常通过将要排序的项目字面上写在卡片上，然后将卡片排成一堆的行为。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/card_sorting/index.md:10]: 1. translate glossary macro
2. some improvements

File: files/zh-cn/glossary/cdn/index.md [hunk 9]
```diff
@@ -1,14 +1,16 @@
...
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-**CDN** (内容分发网络) 指的是一组分布在各个地区的服务器。这些服务器存储着数据的副本，因此服务器可以根据哪些服务器与用户距离最近，来满足数据的请求。CDNs 提供快速服务，较少受高流量影响。
+**CDN** (内容分发网络) 指的是一组分布在各个地区的服务器。这些服务器存储着数据的副本，因此服务器可以根据哪些服务器与用户距离最近，来满足数据的请求。CDN 可以提供快速服务，较少受高流量影响。
 
-CDNs 被广泛用于传输 stylesheets 和 JavaScript 等静态资源，像 Bootstrap，Jquery 等。对这些库文件使用 CDN 技术，有以下几点好处：
+CDNs 被广泛用于传输样式表和 JavaScript 文件等库（例如 Bootstrap、jQquery 等）的静态资源。对这些库文件使用 CDN 技术，有以下几点好处：
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/cdn/index.md:12]: 1. fix incorrect camelcase for word jquery
2. translate stylesheets
3. fix incorrect order of words (static files of library)

File: files/zh-cn/glossary/certified/index.md [hunk 10]
```diff
@@ -1,14 +1,16 @@
...
-**已认证标识**意味着一个程序、一项内容或一次数据传输已经成功经过了相关领域专家的认证，因此该项目被认为是完整的、安全的、可信的。
+**已认证**意味着一个程序、一项内容或一次数据传输已经成功经过了相关领域专家的认证，因而我们认为它是完整的、安全的、可信的。
 
-关于{{glossary("密码学")}}中证书的细节，请参考{{glossary("数字认证证书")}}。
+关于{{glossary("Cryptography", "密码学")}}中证书的细节，请参考{{glossary("Digital Certificate", "数字认证证书")}}。
 
 ## 参见
 
-- 维基百科中的[证书](https://zh.wikipedia.org/wiki/專業認證)
+- 维基百科中的[认证](https://zh.wikipedia.org/wiki/專業認證)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/certified/index.md:16]: give it a name

File: files/zh-cn/glossary/character_encoding/index.md [hunk 11]
```diff
@@ -1,11 +1,13 @@
...
+title: 字符编码
 slug: Glossary/Character_encoding
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-一套编码系统定义字节与文本间的映射。一连串字节文本能让不同文本解释得以进行。我们指明一套特定编码系统时（如 UTF-8），也就指明了字节得以解释的方式。
+一套编码定义字节与文本间的映射。一连串字节文本能让不同文本解释得以进行。我们指明一套特定编码时（如 UTF-8），也就指明了字节得以解释的方式。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/character_encoding/index.md:10]: give it a name

File: files/zh-cn/glossary/character_encoding/index.md [hunk 12]
```diff
@@ -17,5 +19,5 @@ slug: Glossary/Character_encoding
 
 ## 参见
 
-- [Character encoding on W3C](https://www.w3.org/International/articles/definitions-characters/)
+- [W3C 上的字符编码](https://www.w3.org/International/articles/definitions-characters/)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/character_encoding/index.md:22]: translate link name

File: files/zh-cn/glossary/character_set/index.md [hunk 13]
```diff
@@ -1,20 +1,22 @@
...
 
-如果一个字符集使用不正确（例如，对于以 Big5 编码的文章使用 Unicode），你可能会看到全是乱码，这被称为[乱码](https://zh.wikipedia.org/wiki/亂碼)。
+如果一个字符集使用不正确（例如，对以 Big5 编码的文章使用 Unicode），你可能会看到全是不正确的字符，这被称为[乱码](https://zh.wikipedia.org/wiki/亂碼)。
 
 ## 参见
 
-- [字符编码](https://zh.wikipedia.org/wiki/字符编码)（维基百科）
-- [乱码](https://zh.wikipedia.org/wiki/亂碼)（维基百科）
+- 维基百科上的[字符编码](https://zh.wikipedia.org/wiki/字符编码)
+- 维基百科上的[乱码](https://zh.wikipedia.org/wiki/亂碼)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/character_set/index.md:19]: give it a name

File: files/zh-cn/glossary/chrome/index.md [hunk 14]
```diff
@@ -1,11 +1,13 @@
...
 title: Chrome
 slug: Glossary/Chrome
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-在浏览器中，chrome 指除了网页本身以外任何可视的部分（例如：{{glossary("UI")}}、工具栏、菜单栏、标签）。不要将这个概念与 {{glossary("Google Chrome")}} 浏览器混淆。
+在浏览器中，Chrome 指除了网页本身以外任何可视的部分（如 {{glossary("UI")}}、工具栏、菜单栏、标签）。不要将这个术语与 {{glossary("Google Chrome")}} 浏览器混淆。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/chrome/index.md:10]: lint. chrome is a glossary, thus needs to be capitalized.

File: files/zh-cn/glossary/cipher/index.md [hunk 15]
```diff
@@ -1,19 +1,21 @@
 ---
-title: 密码
+title: 加密算法
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/cipher/index.md:2]: should be 加密算法（infered from content below, and from cipher_suite's references)

File: files/zh-cn/glossary/cipher/index.md [hunk 16]
```diff
@@ -24,7 +26,7 @@ slug: Glossary/Cipher
 - [加密与解密](/zh-CN/docs/Archive/Security/Encryption_and_Decryption)
 - [MDN 术语表](/zh-CN/docs/Glossary)
 
-  - {{Glossary("Block cipher mode of operation")}}
+  - {{Glossary("Block cipher mode of operation", "分组密码工作模式")}}
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/cipher/index.md:29]: add recent translated name

File: files/zh-cn/glossary/cipher_suite/index.md [hunk 17]
```diff
@@ -1,18 +1,20 @@
...
+密码套件是密钥交换算法、认证算法、大容量数据{{Glossary("cipher","加密算法")}}和消息认证码算法的组合。
 
-在 {{Glossary("TLS")}} 的{{Glossary("cryptosystem","密码体制")}}中，客户端与服务端在安全通信之前，需要协商出密码套件，一个典型的密码套件类似于 ECDHE_RSA_WITH_AES_128_GCM_SHA256 或 ECDHE-RSA-AES128-GCM-SHA256，代表着：
+在 {{Glossary("TLS")}} 的{{Glossary("cryptosystem","密码系统")}}中，客户端与服务端在安全通信之前，需要协商出密码套件，一个典型的密码套件类似于 ECDHE_RSA_WITH_AES_128_GCM_SHA256 或 ECDHE-RSA-AES128-GCM-SHA256，代表着：
 
 - 密钥交换算法使用 ECDHE（椭圆曲线 Diffie-Hellman 密钥交换算法）
 - 认证算法使用 RSA
 - 加密算法使用 AES-128，使用 Galois/Counter Mode (GCM) 作为分组加密的模式
-- 基于散列的消息认证码算法使用 SHA-256
+- 基于散列的消息认证码（HMAC）算法使用 SHA-256
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/cipher_suite/index.md:17]: some improvements

File: files/zh-cn/glossary/ciphertext/index.md [hunk 18]
```diff
@@ -1,12 +1,14 @@
...
 title: 密文
 slug: Glossary/Ciphertext
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-在{{glossary("cryptography","密码学")}}中，密文是被打乱的信息，在由正确的{{glossary("cipher","密码")}}和正确的密文（通常是{{glossary("key", "密钥")}}）{{glossary("decryption","解密")}}生成原始的{{glossary("Plaintext","明文")}}之前处于不可读状态。密文的安全性及其包含信息的保密性依赖于加密算法和密钥的安全性。
+在{{glossary("cryptography","密码学")}}中，密文是被打乱的信息，在由正确的{{glossary("cipher","密码")}}和正确的密码（通常是{{glossary("key", "密钥")}}）{{glossary("decryption","解密")}}生成原始的{{glossary("Plaintext","明文")}}之前处于不可读状态。密文的安全性及其包含信息的保密性依赖于加密算法和密钥的安全性。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/ciphertext/index.md:10]: distinguish secret and ciphertext

File: files/zh-cn/glossary/ciphertext/index.md [hunk 19]
```diff
@@ -1,12 +1,14 @@
...
 
 ## 参见
 
-- 维基百科上的[密文](https://zh.wikipedia.org/wiki/密文)介绍
+- 维基百科上的[密文](https://zh.wikipedia.org/wiki/密文)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/ciphertext/index.md:14]: lint

File: files/zh-cn/glossary/class/index.md [hunk 20]
```diff
@@ -1,15 +1,18 @@
 ---
-title: Class（类）
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/class/index.md:2]: title improvement

File: files/zh-cn/glossary/class/index.md [hunk 21]
```diff
@@ -1,15 +1,18 @@
...
+title: 类
 slug: Glossary/Class
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-在面向对象编程中（{{glossary("OOP","object-oriented programming")}}）, 一个 _类_ 定义了一个对象（{{glossary("object","object's")}}）的特征。类是定义对象属性（{{glossary("property","properties")}}）和方法（{{glossary("method","methods")}}）的模板，是用来绘制具体对象实例的“蓝图”.
+在{{glossary("OOP", "面向对象程序设计")}}中, 一个*类*定义了一个{{glossary("object", "对象")}}的特征。类是定义{{glossary("property", "属性")}}和{{glossary("method", "方法")}}的模板，是用来绘制具体对象实例的“蓝图”。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/class/index.md:10]: macro improvements

File: files/zh-cn/glossary/class/index.md [hunk 22]
```diff
@@ -1,15 +1,18 @@
...
 ## 参见
 
-- [Class-based vs. prototype-based programming languages](/zh-CN/docs/Web/JavaScript/Guide/Details_of_the_Object_Model#Class-based_vs._prototype-based_languages) (like JavaScript)
-- [Using functions as classes in JavaScript](/zh-CN/docs/Learn/JavaScript/Objects#The_Class)
-- [Class-based programming](https://en.wikipedia.org/wiki/Class-based_programming) on Wikipedia
-- [Object-oriented programming](https://en.wikipedia.org/wiki/Object-oriented_programming) on Wikipedia
+- [JavaScript 中的类](/zh-CN/docs/Learn/JavaScript/Objects/Classes_in_JavaScript)
+- [继承和原型链](/zh-CN/docs/Web/JavaScript/Inheritance_and_the_prototype_chain)
+- 维基百科上的[基于类编程](https://zh.wikipedia.org/wiki/基于类编程)
+- 维基百科上的[面向对象程序设计](https://zh.wikipedia.org/wiki/面向对象程序设计)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/class/index.md:17]: sync with upstream

File: files/zh-cn/glossary/decryption/index.md [hunk 23]
```diff
@@ -5,7 +5,7 @@ slug: Glossary/Decryption
 
 {{GlossarySidebar}}
 
-在密码学（{{glossary("cryptography")}}）领域，**解密**是指把密文（{{glossary("ciphertext")}}）转换成明文（{{glossary("Plaintext")}}）的过程。
+在密码学（{{glossary("cryptography")}}）领域，**解密**是指把{{glossary("ciphertext", "密文")}}转换成{{glossary("Plaintext", "明文")}}的过程。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/decryption/index.md:8]: macro improvement

File: files/zh-cn/glossary/encryption/index.md [hunk 24]
```diff
@@ -5,7 +5,7 @@ slug: Glossary/Encryption
 
 {{GlossarySidebar}}
 
-在密码学（{{glossary("cryptography")}}）领域，**加密**是指把明文（{{glossary("plaintext")}}）转换成密文（{{glossary("ciphertext")}}）的过程。密文应该对非授权用户不可读。
+在密码学（{{glossary("cryptography")}}）领域，**加密**是指把{{glossary("plaintext","明文")}}转换成{{glossary("ciphertext", "密文")}}的过程。密文应该对非授权用户不可读。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/encryption/index.md:8]: macro improvement

File: files/zh-cn/glossary/camel_case/index.md (hunk 7 above)
Comment (yin1999) [reply files/zh-cn/glossary/camel_case/index.md:23]: 原来的链接有什么问题：https://github.com/mdn/content/blob/main/files/en-us/glossary/camel_case/index.md?plain=1

File: files/zh-cn/glossary/cache/index.md (hunk 3 above)
Comment (yin1999) [reply files/zh-cn/glossary/cache/index.md:10]: 全角括号

File: files/zh-cn/glossary/cipher_suite/index.md [hunk 25]
```diff
@@ -1,18 +1,20 @@
...
 title: 密码套件
 slug: Glossary/Cipher_suite
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-密码套件是密钥交换算法、认证算法、大容量数据加密{{Glossary("cipher","加密算法")}}和消息认证码算法的组合。
+密码套件是密钥交换算法、认证算法、大容量数据{{Glossary("cipher","密码算法")}}和消息认证码算法的组合。
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/cipher_suite/index.md:10]:
```suggestion
**密码套件**是密钥交换算法、认证算法、大容量数据{{Glossary("cipher","加密算法")}}和消息认证码算法的组合。
```

File: files/zh-cn/glossary/ciphertext/index.md [hunk 26]
```diff
@@ -1,12 +1,14 @@
...
 title: 密文
 slug: Glossary/Ciphertext
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-在{{glossary("cryptography","密码学")}}中，密文是被打乱的信息，在由正确的{{glossary("cipher","密码")}}和正确的密文（通常是{{glossary("key", "密钥")}}）{{glossary("decryption","解密")}}生成原始的{{glossary("Plaintext","明文")}}之前处于不可读状态。密文的安全性及其包含信息的保密性依赖于加密算法和密钥的安全性。
+在{{glossary("cryptography","密码学")}}中，密文是被打乱的信息，在由正确的{{glossary("cipher","密码算法")}}和正确的密码（通常是{{glossary("key", "密钥")}}）{{glossary("decryption","解密")}}生成原始的{{glossary("Plaintext","明文")}}之前处于不可读状态。密文的安全性及其包含信息的保密性依赖于加密算法和密钥的安全性。
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/ciphertext/index.md:10]:
```suggestion
在{{glossary("cryptography","密码学")}}中，**密文**是被打乱的信息，在由正确的{{glossary("cipher","密码算法")}}和正确的密码（通常是{{glossary("key", "密钥")}}）{{glossary("decryption","解密")}}生成原始的{{glossary("Plaintext","明文")}}之前处于不可读状态。密文的安全性及其包含信息的保密性依赖于加密算法和密钥的安全性。
```

File: files/zh-cn/glossary/class/index.md [hunk 27]
```diff
@@ -1,15 +1,17 @@
...
+title: 类
 slug: Glossary/Class
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-在面向对象编程中（{{glossary("OOP","object-oriented programming")}}）, 一个 _类_ 定义了一个对象（{{glossary("object","object's")}}）的特征。类是定义对象属性（{{glossary("property","properties")}}）和方法（{{glossary("method","methods")}}）的模板，是用来绘制具体对象实例的“蓝图”.
+在{{glossary("OOP", "面向对象程序设计")}}中，一个*类*定义了一个{{glossary("object", "对象")}}的特征。类是定义{{glossary("property", "属性")}}和{{glossary("method", "方法")}}的模板，是用来绘制具体对象实例的“蓝图”。
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/class/index.md:10]:
```suggestion
在{{glossary("OOP", "面向对象程序设计")}}中，一个**类**定义了一个{{glossary("object", "对象")}}的特征。类是定义{{glossary("property", "属性")}}和{{glossary("method", "方法")}}的模板，是用来绘制具体对象实例的“蓝图”。
```

File: files/zh-cn/glossary/cms/index.md [hunk 28]
```diff
@@ -1,12 +1,14 @@
...
 title: CMS
 slug: Glossary/CMS
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-**CMS**（内容管理系统）是一个允许用户发布、组织、修改、删除多种类型的内容的软件。**CMS** 不仅支持文本，还可以嵌入图片、视频、音频和互动的代码。
+CMS（内容管理系统）是一个允许用户发布、组织、修改、删除多种类型的内容的软件。CMS 不仅支持文本，还可以嵌入图片、视频、音频和互动的代码。
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/cms/index.md:10]:
```suggestion
内容管理系统（CMS）是一个允许用户发布、组织、修改、删除多种类型的内容的软件。CMS 不仅支持文本，还可以嵌入图片、视频、音频和互动的代码。
```

File: files/zh-cn/glossary/decryption/index.md (hunk 23 above)
Comment (JasonLamv-t) [reply files/zh-cn/glossary/decryption/index.md:8]:
```suggestion
在{{glossary("cryptography", "密码学")}}领域，**解密**是指把{{glossary("ciphertext", "密文")}}转换成{{glossary("Plaintext", "明文")}}的过程。
```

File: files/zh-cn/glossary/decryption/index.md [hunk 29]
```diff
@@ -5,7 +5,7 @@ slug: Glossary/Decryption
...
 
 解密是一个密码学原语：它通过一种称作{{glossary("cipher")}}的编码技术，把加密信息转换成纯文本。正如加密一样，解密在当代密码学领域也是通过特有的算法，结合称作{{glossary("key")}}的密钥来工作。由于算法通常是公开的，若要保证加密安全，就必须确保密钥高度保密。
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/decryption/index.md:10]:
```suggestion
解密是一个密码学原语：它通过一种称作{{glossary("cipher", "密码算法“)}}的编码技术，把加密信息转换成密文。正如加密一样，解密在当代密码学领域也是通过特有的算法，结合称作 {{glossary("key")}} 的密钥来工作。由于算法通常是公开的，若要保证加密安全，就必须确保密钥高度保密。
```

File: files/zh-cn/glossary/decryption/index.md [hunk 30]
```diff
@@ -5,7 +5,7 @@ slug: Glossary/Decryption
...
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/decryption/index.md:11]: Please translate the image description and the see also section below.

File: files/zh-cn/glossary/encryption/index.md [hunk 31]
```diff
@@ -5,7 +5,7 @@ slug: Glossary/Encryption
...
 
 加密是一个密码学原语：它通过一种称作 {{glossary("cipher")}} 的编码技术，把纯文本信息转换为密文。现代密码的加密算法是通过特定的密码算法来工作的，称为{{glossary("key","密钥")}}。由于算法常常是公开的，若要保证加密安全，就必须确保密钥高度保密。
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/encryption/index.md:10]:
```suggestion
加密是一个密码学原语：它通过一种称作{{glossary("cipher", "密码算法")}}的编码技术，把纯文本信息转换为密文。现代密码的加密算法是通过特定的密码算法，称结合称作 {{glossary("key")}} 的密钥来工作。由于算法常常是公开的，若要保证加密安全，就必须确保密钥高度保密。
```

File: files/zh-cn/glossary/encryption/index.md [hunk 32]
```diff
@@ -5,7 +5,7 @@ slug: Glossary/Encryption
...
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/encryption/index.md:11]: Translate the picture description, same.

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Some suggestions

File: files/zh-cn/glossary/decryption/index.md (hunk 30 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/decryption/index.md:11]: link in the seealso section has a problem. will have a fix in the upstrream source.
Screenshot 2024-05-09 at 7 14 44 PM

File: files/zh-cn/glossary/cia/index.md [hunk 33]
```diff
@@ -9,4 +11,4 @@ CIA（机密性 Confidentiality、完整性 Integrity、可用性 Availability
```
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/glossary/cia/index.md:11]: hah?

```markdown
---
title: CIA
slug: Glossary/CIA
page-type: glossary-definition
---

{{GlossarySidebar}}

**CIA (Confidentiality, Integrity, Availability)** (also called the CIA triad or AIC triad) is a model that guides an organization's policies for information security.

## See also

- [CIA](https://en.wikipedia.org/wiki/Information_security#Key_concepts) on Wikipedia
```

File: files/zh-cn/glossary/character/index.md [hunk 34]
```diff
@@ -11,6 +13,6 @@ slug: Glossary/Character
```
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/glossary/character/index.md:13]: whaat?

File: files/zh-cn/glossary/cacheable/index.md (hunk 4 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/cacheable/index.md:10]: 要不就改成两个分句

**Review by yin1999 — COMMENTED** [review]
fix build

File: files/zh-cn/glossary/decryption/index.md [hunk 35]
```diff
@@ -5,14 +5,17 @@ slug: Glossary/Decryption
...
-![The decryption primitive.](decryption.png)
+![解密原理](decryption.png)
 
 解密是{{glossary("encryption","加密")}}的逆过程，如果密钥高度保密，在没有特定密钥的前提下从数学计算的角度来看，解密是很难做到的。这就取决于算法有多坚固，以及{{glossary("cryptanalysis","密码分析学")}}发展到了什么程度。
 
 ## 参见
 
 - [加密与解密](/zh-CN/docs/Encryption_and_Decryption)
+- {{glossary("Encryption", "加密")}}
+- {{glossary("Cipher, "密码算法")}}
```
Comment (yin1999) [inline files/zh-cn/glossary/decryption/index.md:20]:
```suggestion
- {{glossary("Cipher", "密码算法")}}
```

File: files/zh-cn/glossary/character_set/index.md [hunk 36]
```diff
@@ -1,20 +1,22 @@
...
 - [术语表](/zh-CN/docs/Glossary)
 
   - {{Glossary("Character")}}
```
Comment (yin1999) [inline files/zh-cn/glossary/character_set/index.md:22]:
```suggestion
  - {{Glossary("Character", "字符")}}
```

File: files/zh-cn/glossary/cms/index.md [hunk 37]
```diff
@@ -1,12 +1,14 @@
...
 title: CMS
 slug: Glossary/CMS
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
 ---
 
 {{GlossarySidebar}}
 
-**CMS**（内容管理系统）是一个允许用户发布、组织、修改、删除多种类型的内容的软件。**CMS** 不仅支持文本，还可以嵌入图片、视频、音频和互动的代码。
+内容管理系统（CMS）是一个允许用户发布、组织、修改、删除多种类型的内容的软件。CMS 不仅支持文本，还可以嵌入图片、视频、音频和互动的代码。
```
Comment (JasonLamv-t) [inline files/zh-cn/glossary/cms/index.md:10]:
```suggestion
**内容管理系统（CMS）**是一个允许用户发布、组织、修改、删除多种类型的内容的软件。CMS 不仅支持文本，还可以嵌入图片、视频、音频和互动的代码。
```

**Review by JasonLamv-t — COMMENTED** [review]
Some nits.

File: files/zh-cn/glossary/cms/index.md (hunk 37 above)
Comment (yin1999) [reply files/zh-cn/glossary/cms/index.md:10]: @JasonLamv-t 注意，这样是没法渲染粗体的，请把 `**` 移至左括号前

Comment (JasonLamv-t) [reply files/zh-cn/glossary/cms/index.md:10]: > @JasonLamv-t 注意，这样是没法渲染粗体的，请把 `**` 移至左括号前

I have already updated the suggestion.

File: files/zh-cn/glossary/character/index.md (hunk 34 above)
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/glossary/character/index.md:13]: oops, thx:)

**Review by JasonLamv-t — COMMENTED** [review]
LGTM. Thank you! @PassionPenguin 👍🏼

**Review by JasonLamv-t — APPROVED** [review]

**Pull request merged**